bundles/CoreBundle/src/Migrations/Version20221222134837.php

```diff
@@ -0,0 +1,43 @@
+<?php
+declare(strict_types=1);
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+use Pimcore\Model\Tool\SettingsStore;
+
+/**
+ * Staticroutes will be installed by default
+ */
+final class Version20221222134837 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Install static routes bundle by default';
+    }
+
+    public function up(Schema $schema): void
```

Comment:
please implement a down method which does nothing...the downgrade migration throws error without `down` method implementation:
![image](https://user-images.githubusercontent.com/5137917/211537146-276d0ed6-dfa2-4813-ad52-7a6f7c2d7ac6.png)